components/feedback/wteg_feedback_widget.react.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import WTEGFeedbackForm from './child_components/form/wteg_feedback_form/wteg_feedback_form.react';
+
+import './style.scss';
+
+class WTEGFeedbackWidget extends Component {
+	render() {
+		return (
+			<React.Fragment>
+				<h2>Feedback</h2>
+				<WTEGFeedbackForm />
```

Comment:
As discussed, give the a `prop` eg: `<WTEGFeedbackForm id='what-to-expect-form' />`

Reply:
Done